ui/app/AppLayouts/Communities/views/EditCommunityTokenView.qml

```diff
@@ -380,6 +389,9 @@ StatusScrollView {
             StatusRegularExpressionValidator {
                 id: regexValidatorItem
                 regularExpression: Constants.regularExpressions.alphanumericalExpanded
+                onErrorMessageChanged: {
+                    customInput.validate();
```

Comment:
Why the extra validation step here?

Reply:
because it's validating only when checking the validation rules, in this case we want to change the text in an already "validated" rule